one.js

```diff
@@ -0,0 +1,32 @@
+var gender = prompt("are you a boy or a girl?");
```

Comment:
Looks like you are re-using some of the same code in this program-how could we use functions to encapsulate our code?